ca_description/urdf/color_sensor.xacro

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" ?>
```

Comment:
Change the target of this PR and instead of RoboticaUtnFrba:kinetic-devel, target to your PR where you added the color sensor.

Reply:
I'm not sure how to do this, I can change it with the other branches of the rep but I don't know how to do it with the PR

Reply:
You need to target the branch from which you created a PR. Every PR goes from one branch to another. See for example the screenshot below:
![image](https://user-images.githubusercontent.com/9289769/66501610-133ae200-ea9a-11e9-9adb-f73dd826f52c.png)